### ISSUE TITLE
Scope of the summary guide

### ISSUE THREAD

**Issue by mattgarrish** [w3c/publ-a11y#108]
The title of the guidelines "Accessibility Summary Authoring Guidelines" sounds very generic, like it could be used by anyone for any format to write a summary. The introduction then ties the guide very tightly to EPUB 3.

Writing a guide for anyone is probably outside the scope of this group, although there's no reason why we can't make the guidance as format agnostic as possible, allowing it to be followed for audiobooks, for example. In that case, though, it might be helpful to tack on "for Digital Publications" to the title. But if it's only for EPUB then that should be clearer.

It would also help frame what types of examples we should be depicting - package document, onix, json-ld...

**Comment by GeorgeKerscher** [comment]
I agree with Matt that we need to scope these accessibility Summary guidelines more precisely. I will add "for Digital Publications" to the title and push that PR. I read through the document where this may be expanded and clarified, and noticed we have a heading for Abstract, but the only thing under that is the status of this document. It seems that in an Abstract we could clarify the scope. I do think audio books should be in scope. I also wonder if we should talk about "Optimized Publications."  Regarding Optimized Publications, we know that DAISY books of various types, BRF, Word files, and books accessed through smart assistances/speakers might be considered a variant. These could be TTS or human narration. I would hope the guidelines could be used by the DAISY libraries and other groups serving persons with disabilities as well as mainstream retailers and libraries.

**Comment by murata2makoto** [comment]
Although I also think that the scope should be made more precise, I am not sure if we should cover formats other than EPUB.

First, the relationship between machine-readable schema.org metadata and accessibility summaries has to be considered more carefully.  I personally think that some ebook stores will generate user-friendly descriptions from machine-readable accessibility metadata depending on which accessibility mechanism the EPUB RS of the ebook store provides.   By extending the scope of this document, will we pay less attention to such interactions of machine-readable metadata and accessibility summaries?

Second, some document formats already have mechanisms for in-band metadata and their own accessibility considerations.  Wearing the hat of the convenor of ISO/IEC JTC1/SC34/WG (OOXML), I would like to point out that ISO/IEC 29500-1 already has Annex J. Accessibility Best Practices and ISO/IEC 29500-2 provides OOXML-specific metadata, which can be used to represent accessibility metadata.  I know that there are some accessibility considerations in the ODF specification (both OASIS and ISO/IEC).  I do not know what PDF does, but I won't be surprised if the ISO/IEC PDF standard covers accessibility considerations. If we really want to extend the scope of this document, we should contact committees responsible for such document formats.

**Comment by GeorgeKerscher** [comment]
This issue is refering to:
[The Schema Accessibility Summary](https://www.w3.org/publishing/a11y/schema-a11y-summary/)

It seems The status of this specification should be updated and point to the new Accessibility Metadata Display Guide for Digital Publications 2.0.

I think we address what is mentioned in this issue.
